src/views/List/List.test.jsx

```diff
@@ -0,0 +1,26 @@
+// test list filtering by nation
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import CharacterList from '../List/List';
+
+
+describe('Character List', () => {
+  it('should render a list of characters that are filterable by dropdown', async () => {
+    // render
+    render(<CharacterList />);
+
+    // loading remove
+    await waitForElementToBeRemoved(screen.getByAltText(/avatar wheel of punishment spinner/i));
+
+    // // find dropdown
+    const dropdown = screen.getByRole('combobox');
+
+    // filter with dropdown
+    userEvent.selectOptions(screen.getByRole('combobox'), 'Fire Nation');
+    expect(screen.getByRole('option', {name: 'Fire Nation'}).selected).toBe(true);
+
+    // expect name/character
+    const result = await screen.findAllByText(/fire nation/i);
+    expect(result[0].textContent).toEqual('Fire Nation');
+  })
```

Comment:
In a behavioral test like this, I would assert that the original list is present and then demonstrate that it changes as expected upon dropdown selection.  You are selecting the dropdown and expecting the correct result, but there is no way of seeing how this information changes from its previous state.